tests/test_pagination.py

```diff
@@ -703,6 +627,145 @@ def test_cursor_pagination(self):
         assert isinstance(self.pagination.to_html(), type(''))
 
 
+class TestCursorPagination(CursorPaginationTestsMixin):
```

Comment:
I tried several ways of testing this; first added another mock object that represented a value queryset object but that added a lot of `if`s into the mock queryset and get_pages; then I ended up duplicating the entire test class but that would have duplicated the tests as well; finally ended up extracting out the tests into a mixin and mixing it into the unit tests and value queryset tests. At this point I'm open to any suggestions.